frontend/src/hooks/api/groups/mutations.tsx

```diff
@@ -0,0 +1,130 @@
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+import { apiRequest } from "@app/config/request";
+
+import { organizationKeys } from "../organization/queries";
+import { groupKeys } from "./queries";
+import { TGroup } from "./types";
+
+export const useCreateGroup = () => {
+    const queryClient = useQueryClient();
+    return useMutation({
+        mutationFn: async ({ 
+            name,
+            slug,
+            role
+        }: { 
+            name: string;
+            slug: string;
+            organizationId: string;
```

Comment:
This file seems to be formatted incorrectly. When I format it in my editor, the whole file shifts. Could you look into this? It may be a local configuration issue with Prettier.

Alternatively, you can run the [Prettier fix command](https://prettier.io/docs/en/cli.html)

Reply:
Will do this in another PR.